### ISSUE TITLE
Installation issues: either error 70 permission denied or a message about missing uSysRegInfo table

### ISSUE THREAD

**Issue by samsquared** [joyfullservice/msaccess-vcs-integration#376]
Although I got this installed successfully on a test machine, I can't figure out how to install this on my active computer.  I extracted the zip file to C:\Users\MYUSERNAME\AppData\Roaming\Microsoft\AddIns.  When I go to Add-in manager in access and try to install it, I get an error saying it's missing the USysRegInfo table.  If I double click the ACCDA file I get a permissions error (70).  What am I doing wrong?

Thanks.

**Comment by joyfullservice** [comment]
We refactored this project a while back to use its own internal installer rather than the legacy `USysRegInfo` table. This gives a number of advantages such as being able to install for a local user without requiring admin permissions.

The standard installation process involves downloading and extracting the `Version Control.accda` file from GitHub, and double-clicking the file to begin the installation. (When the addin is opened directly, it automatically opens the install form.)

A permission denied error can occur if there is another instance of Access that has the addin already loaded. In this case the addin file cannot be overwritten because it is already in use. Best practice is to close all open instances of Access before opening the addin file to install it.

If you are still having an issue with the file being in use, you might check in Task Manager to make sure you don't have a hidden instance of Access that is still running after closing all visible instances.

Of course depending on your current macro security settings, you may need to click something to allow the addin to run when you first open it. The installation process automatically sets up the appropriate permissions for the addin to load and run after installation.

Let me know if you continue to have issues with the installation!

**Comment by samsquared** [comment]
Still getting the same permissions error. Debugging points to the deletefile in this sub and strFile is "C:\Users\Sam2\AppData\Roaming\Microsoft\AddIns\Version Control.accda" (the file I double-clicked!).  Should I be running it from another location?

```
Public Sub DeleteFile(strFile As String, Optional blnForce As Boolean = True)
    Perf.OperationStart "Delete File"
    FSO.DeleteFile strFile, blnForce
    Perf.OperationEnd
End Sub

```
I've checked to make sure there are no instances of MSAccess running.  The machine I used for the test was running MS 2016 and the issue I'm having is on MS 2010 - could that be an issue?

In the properties for the accda file, general tab, bottom, it says the file came from another computer and may be blocked...  I've checked to unblock it but that didn't help.

I'm sure it's something simple I'm screwing up.  This can wait until next week.  Thanks for your speedy reply.

**Comment by joyfullservice** [comment]
Thanks for the additional details! I would just manually delete that file. The addin no longer uses the shared addin folder for security reasons, and instead uses its own dedicated folder in `AppData`. The automated (legacy) removal may be failing because it may require more permissions than Access is currently running under.

Access 2010 is no problem at all. That's actually what I use at home and at work, and what I use for developing this addin.

To recap, I think you probably just need to manually remove the addin file from the legacy location, and let the addin run the self-install process to take care of copying the file to the correct folder, setting the necessary security settings and registry entries. Just make sure you are installing the latest release to take advantage of current bug fixes.

Hope that helps! Have a great weekend!

**Comment by samsquared** [comment]
It's installed!  I started the .accda from a different folder (for some reason I thought it had to be in C:\Users\Sam2\AppData\Roaming\Microsoft\AddIns).

Again, thank you for your speedy assistance and Happy New Year!
